app/data_models/list_store.py

```diff
@@ -120,54 +121,53 @@ def __init__(self, existing_items: Optional[List[Mapping]] = None):
 
         self._is_dirty = False
 
-    def __iter__(self):
-        for list_item in self._lists.values():
-            yield list_item
+    def __iter__(self) -> Iterator:
+        yield from self._lists.values()
 
-    def __getitem__(self, list_name):
+    def __getitem__(self, list_name: str) -> Union[ListModel, Any]:
```

Comment:
Should just be a ListModel. FYI, any time `Any` is return, it makes the other types you return obsolete as Any will supersede all types.

Reply:
Changed to ListModel as return.